### ISSUE TITLE
Upcast not recognized

### ISSUE THREAD

**Issue by ivica7** [Yakindu/solidity-ide#82]
```
address a;
Contract c = Contract(a);   // error
```
looks like upcase is recognized as function call.

**Comment by andreasmuelder** [comment]
This is working with the current master. Anything I am missing?
```
contract Contract {

    function () {
		address a;
		Contract c = Contract(a); 
    }
}
```

**Comment by ivica7** [comment]
I am testing with the most recent version from the update site.

Here is the full source where the problem occurs (line ``Migrations upgraded = Migrations(new_address);``):

```
pragma solidity ^0.4.4;

contract Migrations {
  address public owner;
  uint public last_completed_migration;

  modifier restricted() {
    if (msg.sender == owner) _;
  }

  function Migrations() {
    owner = msg.sender;
  }

  function setCompleted(uint completed) restricted {
    last_completed_migration = completed;
  }

  function upgrade(address new_address) restricted {
    Migrations upgraded = Migrations(new_address);   // ERROR: Wrong number of arguments, expected []
    upgraded.setCompleted(last_completed_migration);
  }
}
```

**Comment by andreasmuelder** [comment]
this bug was fixed with PR #103
